security/csrf.rst

```diff
@@ -176,7 +176,7 @@ attribute on the controller action::
     // ...
 
     #[IsCsrfTokenValid('delete-item', tokenKey: 'token')]
```

Comment:
This was asked in the Symfony blog: https://symfony.com/blog/new-in-symfony-7-1-iscsrftokenvalid-attribute#comment-25462

The `'token'` value in `tokenKey`, should be `_token` instead?

Reply:
@javiereguiluz no, <input type="hidden" name="token" value="{{ csrf_token('delete-item') }}"> The value of the token key is determined by what you name it. By default, it's '_token', and when you override its value, you need to mention it in the attribute. Currently, in the documentation, it's correctly mentioned as #[IsCsrfTokenValid('delete-item', tokenKey: 'token')].

Reply:
I've updated the blog post. Thanks!

Reply:
I've checked.You don't need to mention tokenKey: '_token' because that's the default value. You would only use the second parameter, tokenKey, if it's different from '_token'.

Reply:
@rcsofttech85 I'm sorry 🙏  This is fixed now. Thanks a lot for checking it.